pkg/service/elk_service.go

```diff
@@ -127,6 +129,12 @@ func (k *Kibana) Deploy(globalCtx *GlobalCtx) *executor.ExecuteCtx {
 	}
 	args := spec.GetDockerExecCmd(globalCtx.containerCfg, k.spec.ContainerCfg, k.ContainerName, true, mountPoints...)
 	args = append(args, k.spec.Image)
+
+	if k.spec.ProvisioningTemplate != "" {
+		args = append(args, "&&")
+		args = append(args, "/usr/bin/env -S bash", filepath.Join(k.CheckReadyScriptPath))
```

Comment:
you have a check-ready function, why still need this?

Reply:
thx [0abc458](https://github.com/hstreamdb/deployment-tool/pull/28/commits/0abc4580c438c0e15de46119e42d3ee275582383)